_i18n/de/resources/moneropedia/wallet.md

```diff
@@ -1,18 +1,17 @@
 ---
-terms: ["wallet", "wallets"]
-summary: "A wallet stores the information necessary to send and receive Monero"
+terms: ["wallet", "wallets", "Wallet", "Wallets"]
+summary: "Ein Wallet speichert die zum Senden und Empfangen von Monero notwendigen Informationen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-A Monero account, or wallet, stores the information necessary to send and receive Moneroj.  In addition to sending and receiving, the Monero Wallet software keeps a private history of your transactions and allows you to cryptographically sign messages.  It also includes Monero mining software and an address book.
+Ein Monero-Konto beziehungsweise ein Wallet speichert die Informationen, die zum Senden und Empfangen von Monero benötigt werden. Zusätzlich zum Senden und Empfangen führt und bewahrt die Wallet-Software von Monero eine private Chronik deiner Transaktionen und ermöglicht es dir zudem, Nachrichten kryptografisch zu signieren. Sie enthält außerdem ein Programm zum Minen von Monero und ein Adressbuch.
 
-The term "hot wallet" describes a Monero @account which is connected to the Internet.  You can send funds easily but security is much lower than a cold wallet.  Never store large amounts of cryptocurrency in a hot wallet!
+Ein Spar-Wallet ("Hot-Wallet") ist ein mit dem Internet verbundenes Monero-@Konto. Du kannst Gelder zwar bequem versenden, die Sicherheit ist allerdings sehr viel niedriger als es bei einem Spar-Wallet der Fall ist. Bewahre niemals große Mengen von Kryptowährung in einem Zahlungs-Wallet auf! 
 
-A cold wallet is generated on a trusted device or computer via an @airgap.  If the device is to be reused, the data storage should be securely overwritten.  As soon as a cold wallet is connected to the Internet or its mnemonic phrase or @spend-key is entered on an Internet-connected device, it's no longer "cold" and should be considered "hot".
+Ein Spar-Wallet ("Cold-Wallet") wird durch einen @Airgap auf einem vertrauten Gerät oder Computer erzeugt. Falls das Gerät später wiederverwendet werden soll, sollte der Datenspeicher davor sicher überschrieben werden. Sobald ein Spar-Wallet mit dem Internet verbunden wird oder seine mnemonische Passphrase oder sein @Spend-Key in ein mit dem Internet verbundenes Gerät eingegeben wird, sollte es als Zahlungs-Wallet betrachtet werden.
```

Comment:
```suggestion
Ein Spar-Wallet ("Cold-Wallet") wird durch einen @Airgap auf einem vertrauenswürdigen Gerät oder Computer erzeugt. Falls das Gerät später wiederverwendet werden soll, sollte der Datenspeicher davor sicher überschrieben werden. Sobald ein Spar-Wallet mit dem Internet verbunden wird oder seine mnemonische Passphrase oder sein @Spend-Key in ein mit dem Internet verbundenen Gerät eingegeben wird, sollte es als Zahlungs-Wallet betrachtet werden.
```

Reply:
Are you sure it's "verbundenen"? Reads weird to me, but now I'm confused; why use the genitive here? If we stick to that there should also be a change before (I think): "Sobald [...] sein Spend-Key in [or auf?] einem mit dem Internet verbundenen Gerät eingegeben wird [...]".

Reply:
I'm not sure, but my guts, mom and wife told me it's "verbundenen". Neither of them could tell me why though, sorry :) "verbundenes" reads weird to me. I don't think we need a change in the first part of the sentence..? For me it's just "etwas in ein mit dem Internet verbundenen Gerät eingeben" vs "etwas in ein mit dem Internet verbundenes Gerät eingeben"..

Reply:
Doesn't matter contentwise, true, but it bothers me now.. My guts tell me that the case would be wrong with "verbundenen", but technically I'm clueless, and what's 1:3 guts? :)

> An expert opinion would be interesting

It would be indeed!